xbmc/windowing/windows/WinSystemWin32.h

```diff
@@ -153,6 +153,7 @@ class CWinSystemWin32 : public CWinSystemBase
   virtual bool Restore();
   virtual bool Hide();
   virtual bool Show(bool raise = true);
+  virtual void Resize();
```

Comment:
you can try this https://github.com/afedchin/xbmc/commit/6c7469abab8ce349c9010c17bed77a6bd8a5c8c0